tests/testthat/test-independent-testing-n_subject.R

```diff
@@ -3,16 +3,16 @@ library(dplyr)
 library(tidyr)
 
 test_that("if group = ... is not a factor, throw errors", {
-  expect_error(n_subject(r2rtf_adae$USUBJID, r2rtf_adae$TRTA))
+  expect_error(n_subject(r2rtf_adae$USUBJID, as.character(r2rtf_adae$TRTA)))
```

Comment:
`r2rtf_adae$TRTA` used to be a character vector, so this would throw an error. Now we need to coerce to character to get the correct error.